ci/jenkins/test-vmc.sh

```diff
@@ -179,6 +179,35 @@ function setup_cluster() {
         sed -i "s|${kv[0]}|${kv[1]}|g" ${GIT_CHECKOUT_DIR}/jenkins/out/cluster.yaml
     done < "$cluster_defaults"
 
+    # Count the number of clusters created with this network.
+    # If count equals to 6 (2 round of /test-all), switch another one.
+    cluster_count=$(grep -o -i "one" ${WORKDIR}/utils/CLUSTER_NUMBER | wc -l)
```

Comment:
Not sure if multiple jobs running test-vmc.sh would cause race condition when read/write files under ${WORKDIR}/utils/.

Maybe we can make use of the build number. In Jenkins there should be some way to get the build number as an env variable (like `"${BUILD_NUMBER}"`). Then we make a `clusters` array in bash, and we have `cluster_count` clusters in total. Then it does a mod like `index=$((BUILD_NUMBER %  cluster_count))`. Then we just use the `${clusters$[index]}` cluster.

Reply:
I guess you mean network array. Yes, we cana do that.